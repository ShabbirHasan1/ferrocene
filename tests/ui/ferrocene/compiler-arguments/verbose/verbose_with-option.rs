// check-fail
// compile-flags: --verbose=1

fn main() {}

// ferrocene-annotations: um_rustc_verbose
