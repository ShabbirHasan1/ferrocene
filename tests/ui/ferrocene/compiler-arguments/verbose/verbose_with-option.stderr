error: Option 'verbose' does not take an argument

